pkg/cvo/metrics_test.go

```diff
@@ -2,17 +2,16 @@ package cvo
 
 import (
 	"errors"
-	"sort"
-	"strings"
-	"testing"
-	"time"
-
 	"github.com/davecgh/go-spew/spew"
 	"github.com/prometheus/client_golang/prometheus"
 	dto "github.com/prometheus/client_model/go"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/tools/record"
+	"sort"
```

Comment:
These shouldn't be moved. Are you using anon-standard formatter perhaps.

Reply:
Done